src/saveload/saveload.h

```diff
@@ -335,6 +335,7 @@ enum SaveLoadVersion : uint16 {
 	SLV_RIFF_TO_ARRAY,                      ///< 294  PR#9375 Changed many CH_RIFF chunks to CH_ARRAY chunks.
 
 	SLV_TABLE_CHUNKS,                       ///< 295  PR#9322 Introduction of CH_TABLE and CH_SPARSE_TABLE.
+	SLV_AI_INT64,                           ///< 296  PR#9415 SQInteger is 64bit but was saved as 32bit.
```

Comment:
Should probably be `SLV_SCRIPT_INT64` due to it being more generic than just AIs as the commit message already mentions.